File: Data-Structure/card-game/card.cpp
#include <cstdio>
using namespace std;

const int maxn = 55;
int card[maxn];

int main() {
    for (int i = 2; i <= 52; i++) {
        for(int j = i; j <= 52; j += i) {
            card[j] = !card[j];
            //printf("turn %d\n", j);
        }
    }
    card[53] = 1;
    for (int i = 1; i <= 52 ; i++) {
        if(!card[i]) {
            printf("%d ", i);
        }
    }
    return 0;
}

File: Data_Structure/card-game/card.py
card = []

for i in range(0, 54):
    card.append(0)
    # print(card[i])

for i in range(2, 54):
    for j in range(i, 54, i):
        card[j] = ~card[j]

for i in range(1, 54):
    if card[i] == 0:
        print(i,"",end = "")
